storage/tianmu/core/tianmu_table.cpp

```diff
@@ -978,9 +978,12 @@ int TianmuTable::Insert(TABLE *table) {
       return HA_ERR_FOUND_DUPP_KEY;
     }
   }
+  utils::result_set<void> res;
   for (uint i = 0; i < NumOfAttrs(); i++) {
```

Comment:
TianmuTable::Insert will be called if tianmu_sysvar_insert_delayed is OFF, and data should be inserted to disk synchronously. so using async thread to write the data is incorrect here? am i right?